src/scripts/main.js

```diff
@@ -1735,6 +1761,36 @@
         zt.cm_available = (zt.technic === 16 || zt.mystic === 16) && message.cheese.id === 371;
         message.hunt_details = zt;
     }
+    
+    /**
+     * Report the active buffs and floor details
+     * @param {Object <string, any>} message The message to be sent.
+     * @param {Object <string, any>} user The user state object, when the hunt was invoked (pre-hunt).
+     * @param {Object <string, any>} user_post The user state object, after the hunt.
+     * @param {Object <string, any>} hunt The journal entry corresponding to the active hunt.
+     */
+    function addValourRiftStage(message, user, user_post, hunt) {
+        const attrs = user.environment_atts;
+        message.hunt_details = {
+            // Elexir Rain
+            er: attrs.active_augmentations.er,
+            // Sigil Hunter
+            sh: attrs.active_augmentations.hr,
+            // Secret Research
+            sr: attrs.active_augmentations.sr,
+            // Super Siphon
+            ss: attrs.active_augmentations.ss,
+            // String Stepping
+            st: attrs.active_augmentations.sste,
+            // Ultimate Umbra
+            uu: attrs.active_augmentations.tu,
+            // Prestige is incremented after each eclipse in the current run. Starts at 1
+            prestige: attrs.prestige,
+            // Floor type is 1-8, or 7. 
```

Comment:
? 7 is within 1-8

Reply:
I meant 1-7. There is an `is_at_eclipse` flag, so it is possible when at eclipse the floor not to be increased. For safety I added the flag to details.

Reply:
Checked the state for eclipse, it's `floor_type=8`, so no need to record `is_at_eclipse`